code/game/jobs/job/medical.dm

```diff
@@ -233,6 +234,7 @@
 	idtype = /obj/item/weapon/card/id/medical
 	access = list(access_paramedic, access_medical, access_sec_doors, access_maint_tunnels, access_external_airlocks, access_morgue)
 	minimal_access=list(access_paramedic, access_medical, access_sec_doors, access_maint_tunnels, access_external_airlocks, access_morgue)
+	alt_titles = list("EMT","Medical Assitant")

```

Comment:
They're not a medical assistant though. They're the only person with access to the darn paramedic room and that should be implied.